sklearn/tests/test_pipeline.py

```diff
@@ -1015,3 +1018,40 @@ def test_make_pipeline_memory():
     assert_true(pipeline.memory is None)
 
     shutil.rmtree(cachedir)
+
+
+@pytest.mark.parametrize(['est', 'pattern'], [
+    (Pipeline([('transf', Transf()), ('clf', FitParamT())]),
+     r'''\[Pipeline\].*\(step 1 of 2\) Fitting transf.* elapsed=.*\n'''
+     r'''\[Pipeline\].*\(step 2 of 2\) Fitting clf.* elapsed=.*\n$'''),
+    (Pipeline([('transf', Transf()), ('clf', None)]),
+     r'''\[Pipeline\].*\(step 1 of 1\) Fitting transf.* elapsed=.*\n$'''),
+    (Pipeline([('transf', None), ('mult', Mult())]),
+     r'''\[Pipeline\].*\(step 1 of 1\) Fitting mult.* elapsed=.*\n$'''),
+    (FeatureUnion([('mult1', Mult()), ('mult2', Mult())]),
+     r'''\[FeatureUnion\].*\(step 1 of 2\) Fitting mult1.* elapsed=.*\n'''
+     r'''\[FeatureUnion\].*\(step 2 of 2\) Fitting mult2.* elapsed=.*\n$'''),
+    (FeatureUnion([('mult1', None), ('mult2', Mult()), ('mult3', None)]),
+     r'''\[FeatureUnion\].*\(step 1 of 1\) Fitting mult2.* elapsed=.*\n$'''),
+])
+@pytest.mark.parametrize('method', ['fit', 'fit_transform', 'fit_predict'])
+def test_verbose(est, method, pattern, capsys):
+    try:
+        func = getattr(est, method)
+    except AttributeError:
+        return
+    # XXX: getattr(Pipeline(...), 'fit_transform') is always True
```

Comment:
Can you elaborate on that? That seems strange.

Reply:
`Pipeline` always has a defined the `fit_transform` function even if one of the steps does not can not perform `fit_transform`. 

I updated this comment to be more specific about why the test case was skipped.